src/App.js

```diff
@@ -42,6 +44,9 @@ class App extends React.Component {
     if (!web3Providers.wallet) {
       return
     }
+    this.setState({
+      walletWeb3: new Web3(web3Providers.wallet),
```

Comment:
Shall we export getWeb3 so we can use it from outside? https://github.com/aragon/aragon/blob/master/src/aragonjs-wrapper.js#L29

Reply:
Yes, let's expose that cache!